lib/active_fulfillment/services/amazon_mws.rb

```diff
@@ -429,10 +431,18 @@ def escape(str)
 
     private
 
-    def with_error_handling
-      yield
-    rescue ActiveUtils::ResponseError => e
-      handle_error(e)
+    def with_error_handling(max_retries = 0)
+      retries = 0
+      begin
+        yield
+      rescue ActiveUtils::ResponseError => e
+        if e.response == 503 && retries < max_retries
```

Comment:
Think it's okay to retry on non-503's as well. 5xx in general is probably okay.

Reply:
Yea... 5xx should be fine IMO - though I've only seen 503 in the 5xx range